benchmark/daisy/daisy.jl

```diff
@@ -23,20 +23,20 @@ for name in keys(DAISY_1D)
     RELPREC[name] = Dict()
     dom = DAISY_1D[name]["dom"]
     ref = DAISY_1D[name]["ref"]
-    for m in [2, 5, 10]
-        x = Taylor1(m)
+    for ord in [2, 5, 10]
+        x = TaylorModel1(Taylor1(2*ord), 0..0, Interval(mid(dom)), dom)
         p = DAISY_1D[name]["f"](x)
 
-        RESULTS[name]["order $m"] = BenchmarkGroup()
-        RELPREC[name]["order $m"] = Dict()
+        RESULTS[name]["order $ord"] = BenchmarkGroup()
+        RELPREC[name]["order $ord"] = Dict()
 
-        RESULTS[name]["order $m"]["evaluate"] = @benchmarkable evaluate($p, $dom)
+        RESULTS[name]["order $ord"]["evaluate"] = @benchmarkable evaluate($p, $dom)
```

Comment:
The evaluation has to be computed as differences from the expansion point, that is `evaluate($p, $dom-$(mid(dom)))`, *unless* the mid-point of `dom` is zero.

Reply:
Is this only the case when we use "independent variable"?

Reply:
Taylor expansions are done around a point (call it `t0`), and make sense if you are "close enough" to that point. They are usually written as `x(t) = x0 + x1*(t-t0) + x2*(t-t0)^2 ...`; in TaylorSeries we abuse of notation and skip the `t0` dependence of the series. Yet, you still have to include it for proper evaluation.

Reply:
That explains it thanks.